packages/admin/components/composed/item/ItemLayout/ItemLayout.tsx

```diff
@@ -47,7 +47,7 @@ export default function ItemLayout({
           memoizedItem.title || "glossary.item"
         );
         hideDialog();
-        router.replace(breadcrumbs[breadcrumbs.length - 1]?.href);
+        router.replace(breadcrumbs[breadcrumbs.length - 2]?.href);
```

Comment:
What was the bug with length-1?

Reply:
`breadcrumbs[length-1]` points to the item itself. We want the parent here because this is the redirect after the item is deleted. I think?